roles/splunk/handlers/main.yml

```diff
@@ -15,16 +15,31 @@
   ignore_errors: true
   when: splunk_use_initd and ansible_os_family == "RedHat" and ansible_distribution_major_version|int > 6
 
+- name: update init.d to accept license
+  replace:
+    path: /etc/init.d/splunk
+    regexp: '(start --no-prompt --answer-yes)$'
+    replace: '\1 --accept-license'
+  become: yes
+  when: splunk_use_initd
+
+- name: reload systemctl daemon
+  systemd:
+    daemon_reload: yes
+  become: yes
+  when: ansible_connection!="docker"
```

Comment:
Thinking back to when I tried to use `systemctl` with docker, I wonder if this shouldn't be skipped when used under docker, but instead left up to the admin using this to ensure that their container works with `systemctl`, or alternatively that they use `initd` instead, which would be skipped if this was `when: not splunk_use_initd`.

Reply:
Removed conditional